### PR TITLE
Added Docker Container

### PR DESCRIPTION
**Pull request by arslantoor**

### CODE REVIEW

**Comment by adeelnawaz** [comment]
- PR and commit titles should be self-explanatory e.g. "Dockerized the app" or "Application dockerization", etc
- Why did you copy the whole nginx-php-fpm into your source? You're supposed to create only a Dockerfile with **base image** set to nginx-php-fpm
- .idea folder should be in gitignore
- Why is there a composer.lock file?

**Review by adeelnawaz — CHANGES_REQUESTED** [review]
- PR and commit titles should be self-explanatory e.g. "Dockerized the app" or "Application dockerization", etc
- Why did you copy the whole nginx-php-fpm into your source? You're supposed to create only a Dockerfile with base image set to nginx-php-fpm
- .idea folder should be in gitignore
- Why is there a composer.lock file?
- Add description to your PR

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,71 @@
+
+###> symfony/framework-bundle ###
```
Comment (adeelnawaz) [inline .gitignore]: What is all this in .gitignore?

File: docker-compose.yml.dist [hunk 2]
```diff
@@ -0,0 +1,8 @@
+version: '3.7'
+
+services:
+  custom-container:
```
Comment (adeelnawaz) [inline docker-compose.yml.dist]: Give your service a proper name, such as testapp or test_app or demo_app etc

File: docker-compose.yml.dist [hunk 3]
```diff
@@ -0,0 +1,8 @@
...
+    build:
+      context: .
+      dockerfile: ./nginx/Dockerfile
+    container_name: custom-container
```
Comment (adeelnawaz) [inline docker-compose.yml.dist]: Also container name should be proper

**Review by adeelnawaz — CHANGES_REQUESTED** [review]

File: docker-compose.yml.dist (hunk 3 above)
Comment (arslantoor) [reply docker-compose.yml.dist]: okay

File: .gitignore (hunk 1 above)
Comment (arslantoor) [reply .gitignore]: these are files for symfony

**Review by arslantoor — COMMENTED** [review]
naming convention improve for services in docker-compose.yml.dist

**Pull request merged**